external-api/src/http/mod.rs

```diff
@@ -4,7 +4,7 @@ use serde::{Deserialize, Serialize};
 
 pub mod network;
 pub mod order_book;
-pub mod price_report;
+pub mod price_report; // TODO: REMOVE AFTER FE INTEGRATION W/ STANDALONE PRICE REPORTER
```

Comment:
are we removing this? Re: our discussion this morning, I though we plan to leave it as an option (though with the new paired down interface)? I'll say it's helpful for debugging occasionally

Reply:
Ah, if it's helpful for debugging, I can leave it.
I was thinking we'd just leave access to the relayer's view of the price via the websocket endpoint that allows subscribing to system bus topics, but I can also leave the HTTP endpoint and just have it return the "canonical" price alone instead of also returning the per-exchange prices